src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const longTermDiscount = 50;
+  const shortTermDiscount = 20;
+  const defaultRentalPrice = 40;
+
+  if (days < 3) {
+    return days * defaultRentalPrice;
+  }
+
+  if (days >= 3 && days < 7) {
+    return days * defaultRentalPrice - shortTermDiscount;
+  }
+
+  if (days >= 7) {
+    return days * defaultRentalPrice - longTermDiscount;
```

Comment:
The long-term discount is applied as a flat rate, which is incorrect according to the task requirements. The discount should be applied per day. Consider changing the logic to `return days * (defaultRentalPrice - longTermDiscount);`.